test/deploy-eks-cluster.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+function deleteCluster() {
+  eksctl delete cluster --name=${CLUSTER_NAME}
+  if [ $? -ne 0 ]; then
+    echo "Unable to delete cluster - ${CLUSTER_NAME}"
+    return 1
+  fi
+
+  return 0
+}
+
+function createCluster() {
+  # Deploy eksctl cluster if not deploy
+  rc=$(which eksctl)
+  if [ -z "$rc" ]; then
+    echo "eksctl is not installed or in the PATH. Installing..."
+    os=$(uname -s)
+    curl --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_${os}_amd64.tar.gz" | tar xz -C /tmp
+    if [ $? -ne 0 ]; then
+     echo "Unable to download eksctl for ${os}."
+     return 1
+    fi
+    chmod +x /tmp/eksctl
+    sudo mv /tmp/eksctl /usr/local/bin
```

Comment:
Not sure we should have createCluster() install this versus having it as a dependency for testing on eks clusters?

Reply:
Agree. Install this in circleci config.yml